sdk/ahnlich-client-go/tests/utils_test.go

```diff
@@ -0,0 +1,13 @@
+package ahnlichgotest
```

Comment:
Do we need this file?

Reply:
I don't think so; was using it for sanity check